plugin-animation/src/test/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImplTest.kt

```diff
@@ -1006,31 +1018,30 @@ class CameraAnimationsPluginImplTest {
     shadowOf(getMainLooper()).pause()
     cameraAnimationsPluginImpl.easeTo(cameraState.toCameraOptions(), mapAnimationOptions { duration(DURATION) })
     shadowOf(getMainLooper()).idle()
-    verify(exactly = 0) { logD(TAG, any()) }
+    verify(exactly = 0) { logI(TAG, any()) }
   }
 
   @Test
   fun catchExceptionOnSetCameraTest() {
-    executeSetCameraTest(Exception("Invalid camera options"))
+//    executeSetCameraTest(Exception("Invalid camera options"))
```

Comment:
why do we comment out this test? if not needed, maybe use `@ignore` annotation with some context would be better.

Reply:
Reverted, goot catch